app/views/users/index.html.erb

```diff
@@ -1 +1,6 @@
-<p><%= @users %></p>
\ No newline at end of file
+<%@users.each do |user|%>
+<img src = "<%=user.photo%>">
+<h2><%= user.name%></h2>
+<p>number of posts: <%= user.posts_counter%></p>
+<br>
+<%end%>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design%20from%20the%20sneak%20peek%20wireframes), Please add basic styling to users index view to make it match the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_root_users.png).

- [ ] Clicking on each user suppose to take you to their show page. So please wrap the whole user by `link_to` to do that. See this [article](https://www.rubyguides.com/2019/05/rails-link_to-method/) to understand more.